perfkitbenchmarker/rackspace/rackspace_machine_types.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module contains constants, baked-in assumptions and utility functions related
+ to Rackspace flavors, aka. machine types."""
```

Comment:
This whole module and its functions exists to translate the scratch_disk_types to something 'appropriate', depending on the flavor. Along with some constants that are specific to the machine types. In Rackspace, we support both different virtualized, and bare metal instances, some of the assumptions change across these machine_types, such as block device name format, number of disks included per flavor, etc.

That module is mostly based out of this support doc:
http://docs.rackspace.com/servers/api/v2/cs-devguide/content/server_flavors.html

Once we introduce the changes suggested in #284, then most of it, if not all, will be simplified and we won't need to do such translation.